.stestr.conf

```diff
@@ -1,3 +1,3 @@
 [DEFAULT]
-test_path=${TEST_PATH:-./esiclient/tests}
```

Comment:
Since we use pytest in https://github.com/CCI-MOC/python-esileapclient, we should probably use pytest in this repo too (at least for the functional tests). I know the method I used to mark tests as slow assumed pytest as a test runner (although it may work fine under stestr too) and it would probably be nice regardless for consistency's sake.

Reply:
I changed this because I was having issues with the functional tests trying to run the unit tests. What would be the workaround to this?